index.bs

```diff
@@ -150,7 +213,31 @@ the following steps:
 1. Return |encodedOutput|.
 
 <h3 algorithm id="delivery-time">Establishing report delivery time</h3>
-TODO
+The <dfn>report delivery time</dfn> for an [=impression=] |impression| and a
+[=conversion/conversion time=] |conversionTime| is the result of the following steps:
+1. Let |conversionTimeAfterImpression| be the difference between the
+    [=conversion/conversion time=] and [=impression/impression time=].
+2. Let |expiryDelta| be the difference between the [=impression/expiry=] and
+    the [=impression/impression time=]
+
+<dl class="switch">
+<dt>If |conversionTimeAfterImpression| <= (2 days - 1 hour)</dt>
+<dd>return [=impression/impression time=] + 2 days.</dd>
```

Comment:
Is it intentional that the report delivery time is impression-time + 2 days even if conversion-time > expiry? That might deserve a note.

Reply:
Ah good catch, this shouldn't be possible because after expiry the impression will be deleted. I'll add a note